mifosng-android/src/main/res/values/strings.xml

```diff
@@ -137,6 +137,40 @@
     <string name="update">Update</string>
     <string name="group_clients">Group Clients</string>
     <string name="identifier_deleted_successfully">Identifier deleted Successfully</string>
+    <string name="next">Next</string>
+    <string name="purpose">Purpose</string>
+    <string name="surveyEmpty">This Survey is Empty</string>
+    <string name="charges_loaded_successfully">Charges Loaded Successfully</string>
+    <string name="charge_created_success">Charge created successfully</string>
+    <string name="no_more_charges_available">No more Charges Available</string>
+    <string name="deposit_success__transaction_id">"Deposit Successful, Transaction ID = "</string>
+    <string name="withdraw_success__transation_id">"Withdrawal Successful, Transaction ID = "</string>
+    <string name="thanks_for_taking_survey">Thanks for taking Survey</string>
+    <string name="no_charges_to_show">There is No Charges to Show</string>
+    <string name="center_created">Center created successfully</string>
+    <string name="connected_to_loc_services">Connected to location services</string>
+    <string name="charge_dialog_fragment">Charge Dialog Fragment</string>
+    <string name="disconnected_from_loc_services">Disconnected from location services</string>
+    <string name="image_deleted">Image deleted</string>
+    <string name="loan_submitted_for_approval">The Loan has been submitted for Approval</string>
+    <string name="make_repayment">Make Repayment</string>
+    <string name="approve_loan">Approve Loan</string>
+    <string name="disburse_loan">Disburse Loan</string>
+    <string name="loan_closed">Loan Closed</string>
+    <string name="working">Working...</string>
+    <string name="updating_gps">Updating Gps Data</string>
+    <string name="save_pin">Save pin</string>
+    <string name="survey_name">Survey Name</string>
+    <string name="start_tracking">Start tracking</string>
+    <string name="stop_tracking">Stop tracking</string>
+    <string name="client_id">237</string>
```

Comment:
I don't quite like this string here. 

If this is just for debugging purposes.

You should use ```tools:text``` attribute and you can hardcode your string there